Makefile

```diff
@@ -404,6 +404,10 @@ $(GOPATH)/bin/golangci-lint:
 .PHONY: lint
 lint: server/static/files.go $(GOPATH)/bin/golangci-lint
 	rm -Rf v3 vendor
+	# If you're using `woc.wf.Spec` or `woc.execWf.Status` your code probably won't work with WorkflowTemplate.
+	# * Change `woc.wf.Spec` to `woc.execWf.Spec`.
+	# * Change `woc.execWf.Status` to `woc.wf.Status`.
+	@awk '(/woc.wf.Spec/ || /woc.execWf.Status/) && !/not-woc-misuse/ {print FILENAME ":" FNR "\t" $0 ; exit 1}' $(shell find workflow/controller -type f -name '*.go' -not -name '*test*')
```

Comment:
Can we also add a note in the docstring of `woc.wf` and `woc.execWf` and clarify the differences?

Reply:
I don't feel strongly about this.

Reply:
Yes. I agree I realise.

Reply:
Oh. Actuallly, this is already the case.